consensus/integration/nodes_test.go

```diff
@@ -231,6 +458,36 @@ func createNode(
 	validator := consensus.NewValidator(metrics, committee, forks)
 	require.NoError(t, err)
 
+	keys := &storagemock.DKGKeys{}
+	// there is DKG key for this epoch
+	keys.On("RetrieveMyDKGPrivateInfo", mock.Anything).Return(
```

Comment:
This was changed recently - https://github.com/onflow/flow-go/blob/8a66f3766040457b46290ad3051c80d54c5e487a/storage/badger/dkg_state.go#L143

Reply:
fixed